src/platform/forms-system/src/js/review/ReviewCollapsibleChapter.jsx

```diff
@@ -362,6 +375,7 @@ class ReviewCollapsibleChapter extends React.Component {
                 aria-controls={`collapsible-${this.id}`}
                 onClick={this.props.toggleButtonClicked}
                 id={`collapsibleButton${this.id}`}
+                type="button"
```

Comment:
Does this button actually need `type="button"`? Not sure if this element is embedded in a `<form />` tag.

Reply:
No, but it resolves a linter warning. This button is not within a form element. It's used to toggle the accordion item.